### PR TITLE
dynamic biography from backend

### PR DESCRIPTION
**Pull request by ansh-k**
1 Update dynamic **Biography** from backend 
bookstorehome

bookstoreauthorpage

### CODE REVIEW

File: app/models/author.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { on } from '@ember/object/evented';
 
 export default Publisher.extend({
   name: attr(),
+  biography: attr(),
   books: hasMany(),
```
Comment (vladucu) [inline app/models/author.js]: It seems you are missing some details here, like the `attr` type and the `hasMany` relationship model

**Review by vladucu — COMMENTED** [review]
Generally, here at Smile we always try to include a screen-shot and/or gif with changes for frontend PRs, do you mind doing the same please?

File: app/models/author.js (hunk 1 above)
Comment (ansh-k) [reply app/models/author.js]: @vladucu Updated changes with `attr` as a string and `hasMany`

**Review by vladucu — APPROVED** [review]
Nicely done @ansh-k !!!

Thanks for the images too. This looks pretty good to me, let's merge this in 🎉

**Pull request merged**